<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1236</width>
    <height>966</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*Copyright (c) DevSec Studio. All rights reserved.

MIT License

Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the &quot;Software&quot;), to deal
in the Software without restriction, including without limitation the rights
to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
copies of the Software, and to permit persons to whom the Software is
furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
*/

/*-----QWidget-----*/
QWidget
{
	background-color: #00111d;
	color: #000000;
	border-color: #000000;

}


/*-----QLabel-----*/
QLabel{
    font-family: Consolas, &quot;Courier New&quot;, monospace;
	color: rgb(232, 232, 232);
	font: 20pt;
}

/*-----QPushButton-----*/
QPushButton {
    height: 50px;
    width: 120px;
    background-color: #333;
    border: 1px solid #404c5d;
    color: rgb(232, 232, 232);
    font-family: Consolas, &quot;Courier New&quot;, monospace;
    font-size: 14px;
    border-radius: 5px;

    /* Gradient background */
    background: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #2e2d2d,
        stop:1 #212121
    );

    padding: 5px;
}

/* Hover */
QPushButton:hover {
    background: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #353535,
        stop:1 #262626
    );
    color: #d6d6d6;
}

/* Pressed */
QPushButton:pressed {
    background: qlineargradient(
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #262626,
        stop:1 #1a1a1a
    );
    color: #e0e0e0;

    /* Slight inset effect */
    padding-top: 7px;
    padding-bottom: 3px;
}


/*-----QToolButton-----*/
QToolButton
{
	background-color: #00111d;
	color: #000000;
	border-style: solid;
	border-color: #000000;
	padding: 5px;

}


QToolButton::hover
{
	background-color: #00111d;
	color: #000000;
	border-style: solid;
	border-width: 1px;
	border-color: #01dcff;

}


QToolButton::pressed
{
	background-color: #01dcff;
	color: #000000;
	border-style: solid;
	border-width: 1px;
	border-color: #01dcff;

}


/*-----QLineEdit-----*/
QLineEdit
{
	background-color: #082336;
	color: #3e93ff;
	border-style: solid;
	border-width: 2px;
	border-color: #1e4b6d;
	padding: 10px;

}


QLineEdit::hover
{
	background-color: #082336;
	color: #3e93ca;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}


QLineEdit::focus
{
	
	color: #3e93ca;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}






</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>1381</width>
      <height>871</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Add Student / Delete Student</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>1141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>12</y>
         <width>551</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 24pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Students Add/Remove System</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>1141</width>
        <height>171</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="Tab1_LastName">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>70</y>
         <width>261</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>First Name:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>80</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Last Name :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Tab1_FirstName">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>70</y>
         <width>261</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>310</y>
        <width>1141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="Tab1_AddStudent">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>20</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Tab1_DeleteStudent">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>20</y>
         <width>241</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>410</y>
        <width>1141</width>
        <height>331</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTableWidget" name="Tab1_StudentTable">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>0</y>
         <width>1001</width>
         <height>331</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">    
    QTableWidget {
        background-color: #001622;
        color: white;
        font-size: 14px;
        border: 1px solid #4c4c4c;
    }

    QHeaderView::section {
        background-color: #0b2e4a;
        color: white;
        padding: 5px;
        border: 1px solid #32414B;
        font-weight: bold;
    }

    QTableWidget::item:selected {
        background-color: #1e90ff;
        color: black;
    }
 </string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="cornerButtonEnabled">
        <bool>false</bool>
       </property>
       <column>
        <property name="text">
         <string>Student ID</string>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignLeading|AlignVCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Email Address</string>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Subject Enrollment</string>
     </attribute>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>1381</width>
        <height>711</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>70</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Student ID : </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Tab2_StudentID">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>70</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="Tab2_AddCourse">
       <property name="geometry">
        <rect>
         <x>980</x>
         <y>640</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Tab2_DeleteCourse">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>640</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
      <widget class="QLabel" name="EnrollmentMessege">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>640</y>
         <width>501</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QTableWidget" name="StudentCourseTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>591</width>
         <height>481</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">    
    QTableWidget {
        background-color: #001622;
        color: white;
        font-size: 14px;
        border: 1px solid #4c4c4c;
    }

    QHeaderView::section {
        background-color: #0b2e4a;
        color: white;
        padding: 5px;
        border: 1px solid #32414B;
        font-weight: bold;
    }

    QTableWidget::item:selected {
        background-color: #1e90ff;
        color: black;
    }</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="cornerButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QTableWidget" name="Tab2_SubjectsTable">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>130</y>
         <width>591</width>
         <height>481</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">    
    QTableWidget {
        background-color: #001622;
        color: white;
        font-size: 14px;
        border: 1px solid #4c4c4c;
    }

    QHeaderView::section {
        background-color: #0b2e4a;
        color: white;
        padding: 5px;
        border: 1px solid #32414B;
        font-weight: bold;
    }

    QTableWidget::item:selected {
        background-color: #1e90ff;
        color: black;
    }</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="columnCount">
        <number>0</number>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>80</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Sections Capacites</string>
     </attribute>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Section Code : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Tab3_Section_ID_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>40</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>New Capacity : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Tab3_Capacity">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>40</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="CapacityTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>1351</width>
        <height>521</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    
    QTableWidget {
        background-color: #001622;
        color: white;
        font-size: 14px;
        border: 1px solid #4c4c4c;
    }

    QHeaderView::section {
        background-color: #0b2e4a;
        color: white;
        padding: 5px;
        border: 1px solid #32414B;
        font-weight: bold;
    }

    QTableWidget::item:selected {
        background-color: #1e90ff;
        color: black;
    }</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>Course Code</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Section ID</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Capacity</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="Tab3_Confirm">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>140</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Grading System</string>
     </attribute>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Student ID :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>40</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Course Code : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>40</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Course Grade : </string>
      </property>
     </widget>
     <widget class="QComboBox" name="Tab4_Course_Grade">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>40</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
    background-color: white;
    color: black;
    font-size: 16px;
    font-weight: bold;
    padding: 4px;
}

QComboBox QAbstractItemView {
    background-color: white;
    color: black;
    font-size: 16px;
    font-weight: bold;
}
</string>
      </property>
      <item>
       <property name="text">
        <string>A+</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>A</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B+</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>C+</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>C</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>D+</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>D</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>F</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="Tab4_Student_ID">
      <property name="geometry">
       <rect>
        <x>202</x>
        <y>41</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Tab4_Course_code">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>40</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="StudentGradeTable_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>250</y>
        <width>1241</width>
        <height>441</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    
    QTableWidget {
        background-color: #001622;
        color: white;
        font-size: 14px;
        border: 1px solid #4c4c4c;
    }

    QHeaderView::section {
        background-color: #0b2e4a;
        color: white;
        padding: 5px;
        border: 1px solid #32414B;
        font-weight: bold;
    }

    QTableWidget::item:selected {
        background-color: #1e90ff;
        color: black;
    }</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>Student ID</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Course ID</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Course Name</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Course Grade</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="Tab4_Confirm">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>180</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1236</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
